src/preload/native.ts

```diff
@@ -0,0 +1,16 @@
+import { ipcRenderer } from 'electron'
+
+export default {
+    on: (event, callback) => {
+        ipcRenderer.on('native-event', (_, data) => {
+
+            // Strip leading slashes
+            event = event.replace(/^(\\)+/, '');
+            data.event = data.event.replace(/^(\\)+/, '');
+
+            if(event === data.event) {
+                return callback(data.payload);
```

Comment:
Could be nice to pass the event name into the callback as a second parameter

Reply:
Added! The name passed to the callback does have those leading slashes stripped. Is that alright?

Reply:
Yes, I think this is fine